app/os/main.cpp

```diff
@@ -24,7 +31,12 @@ auto battery	  = leka::CoreBattery {PinName::BATTERY_VOLTAGE, charge_input};
 auto sd_blockdevice = SDBlockDevice {SD_SPI_MOSI, SD_SPI_MISO, SD_SPI_SCK};
 auto fatfs			= FATFileSystem {"fs"};
 
-auto rc = RobotController {sleep_timeout, battery};
+auto coreqspi		   = CoreQSPI();
+auto coremanageris25lp = CoreFlashManagerIS25LP016D(coreqspi);
+auto coreis25lp		   = CoreFlashIS25LP016D(coreqspi, coremanageris25lp);
+auto firmwarekit	   = FirmwareKit(coreis25lp);
```

Comment:
```suggestion
auto coreflashmanager = CoreFlashManagerIS25LP016D(coreqspi);
auto coreflash		   = CoreFlashIS25LP016D(coreqspi, coremanageris25lp);
auto firmwarekit	   = FirmwareKit(coreflash);
```

mais ça fait bizarre d'avoir le manager de la flash qui est passé en argument pour la flash, j'aurais fait l'inverse.

Reply:
Il faudrait revoir nos discussions pour voir ce qui avait motivé à implémenter le Manager dans la Memory et non l'inverse

Reply:
je fais un task, c'est pas urgent et si ça se trouve c'est juste une problème de naming. ;)